dev.yml

```diff
@@ -2,7 +2,7 @@
 name: recordstore
 
 up:
-  - ruby: 2.3.1
+  - ruby: 2.4.3
```

Comment:
Should this be in a different PR or you rely on this version to have things working properly?

Reply:
I can remove... but was small enough I thought you'd let me away without doing a separate PR :-P

Reply:
If you think that won't cause any problem because of the ruby bump, I'm fine with it 👍